packages/kbn-pm/src/utils/bazel/setup_remote_cache.ts

```diff
@@ -64,55 +47,14 @@ export async function setupRemoteCache(repoRootPath: string) {
 
   const settingsPath = resolve(repoRootPath, '.bazelrc.cache');
 
-  // Checks if we should upgrade the servers used on .bazelrc.cache
-  //
-  // NOTE: this can be removed in the future once everyone is migrated into the new servers
-  if (await migrateToNewServersIfNeeded(settingsPath)) {
-    return;
-  }
-
-  if (existsSync(settingsPath)) {
-    log.debug(`[bazel_tools] remote cache settings already exist, skipping`);
-    return;
-  }
-
-  if (!(await isVaultAvailable())) {
-    log.info('[bazel_tools] vault is not available, unable to setup remote cache settings.');
-    log.info('[bazel_tools] building packages will work, but will be slower in many cases.');
-    log.info('[bazel_tools] reach out to Operations if you need assistance with this.');
-    return;
-  }
-
-  let apiKey = '';
-
-  try {
-    const { stdout } = await spawn(
-      'vault',
-      ['read', '-field=readonly-key', 'secret/ui-team/kibana-bazel-remote-cache'],
-      {
-        stdio: 'pipe',
-      }
-    );
-    apiKey = stdout.trim();
-  } catch (ex: unknown) {
-    log.info(
-      '[bazel_tools] unable to read bazel remote cache key from vault, are you authenticated?'
-    );
-    log.info('[bazel_tools] building packages will work, but will be slower in many cases.');
-    log.info('[bazel_tools] reach out to Operations if you need assistance with this.');
-    log.info(`[bazel_tools] ${ex}`);
-
+  if (!upToDate(settingsPath)) {
+    log.debug(`[bazel_tools] remote cache settings already up to date, skipping`);
     return;
   }
 
   const contents = dedent`
-    # V1 - This file is automatically generated by 'yarn kbn bootstrap'
+    # 7.17 V2 - This file is automatically generated by 'yarn kbn bootstrap'
     # To regenerate this file, delete it and run 'yarn kbn bootstrap' again.
-    build --bes_results_url=https://app.buildbuddy.io/invocation/
```

Comment:
It's not really clear to me if this branch if gcs ready.  If it is we can probably use the global V2.